### PR TITLE
Replace 'places' verbiage with 'cities' for consistency

### LINKED ISSUE #296: "Cities" vs "Places"
Currently inconsistent verbiage. I recommend using "Cities". That is easier for me to wrap my head around. I don't immediately know what "Places" means in the nav bar.

### PR DESCRIPTION
**Pull request by mehulkar**
Closes #296.

### CODE REVIEW

File: app/views/shared/_header.html.haml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 %nav.header
   %h1.logo= link_to 'Tea With Strangers', root_path
   %ul.nav-list
-    %li= link_to "Places", cities_path
+    %li= link_to "Cities", cities_path
```
Comment (nickbarnwell) [inline app/views/shared/_header.html.haml]: @ankitshah811 to weigh in

**Comment by mehulkar** [comment]
Screenshot of what it looks like after this change: (nav says "Cities" instead of "Places")

screen shot 2015-04-24 at 10 50 57 am

**Comment by mehulkar** [comment]
@nickbarnwell I changed 2 files in this PR. It looks like `_nav_header.html.html` is being used in places, but I can't tell is `_header.html.html` is being used anywhere. Any idea?

**Comment by ankitshah811** [comment]
Wow cities makes so much more sense thank you @mehulkar

**Pull request merged**